index.html

```diff
@@ -818,7 +818,30 @@ <h2>Help your friends find some good learning resources</h2>
                 </div>
             </div>
             <!--Deepanshu's card End--!>
-            
+             <!--Sneha's card-->
+            <div class="card">
+                <p class="name">Sneha Agarwal</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://twitter.com/isyneha" target="_blank">@isyneha</a>
+                </p>
+                <p class="about">I am a pen-ultimate third year undergraduate at HArcourt Butler Technical University, Kanpur. A firm believer who loves to explore various domains.</p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a href="https://www.w3schools.com" target="_blank" title="First Resource">Site for learning front-End Development</a>
+                        </li>
+                        <li>
+                            <a href="https://www.freecodecamp.org/learn" target="_blank" title="Second Resource">Another good course for Front-end Developemnt and more</a>
+                        </li>
+                        <li>
+                            <a href="https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwj65rbt6c36AhWbCysKHT4jB10YABAAGgJzZg&ohost=www.google.com&cid=CAESbOD2UAJMFhM9J31yrM5ODOyKWzAsSkGhg4dw_9nryhahPC9aHo5pHiFv_y4Fo-RBpPqCCo4b_89ylU2yBsAM-cMdHY4fQFBAa5HUH1vTPHW42nVxcvhvarSQbXsDGehWUwFzdzw835J_zKceXA&sig=AOD64_3xN71o26XxcBVItcFKt5pHm9ZCgg&q&adurl&ved=2ahUKEwipjbLt6c36AhWIXWwGHe_IDFQQ0Qx6BAgIEAE" target="_blank" title="Third resource">Edit PDF file online</a>
```

Comment:
Hey is this your google ad id and page or any thing related to it ?

Reply:
> Hey is this your google ad id and page or any thing related to it ?

line no 839

Reply:
also all the things after getting actual link in url there are so many parms like account and id location

Reply:
I have corrected it. Please have a look @Rajkumar-justcoder

Reply:
> I have corrected it. Please have a look @Rajkumar-justcoder

can u just replace link with other links 

why i am telling is after reading this : https://sourceforge.net/software/product/PDFSimpli/

Reply:
Sure